addons/account/wizard/account_move_send.py

```diff
@@ -227,6 +245,13 @@ def _get_mail_template_attachments_data(self, mail_template):
     # COMPUTE METHODS
     # -------------------------------------------------------------------------
 
+    @api.depends('move_ids')
+    def _compute_show_pdf_template_menu(self):
+        available_templates_count = self.env['ir.actions.report'].search_count([('is_invoice_report', '=', True)], limit=2)
```

Comment:
I'm afraid about the multi-company here. To check functionaly

Reply:
I tested with multi-company and had no issues

Reply:
That's weird in some multi-company flows but that's ok